conn_test.go

```diff
@@ -1687,3 +1687,87 @@ func TestProtocolVersionValidation(t *testing.T) {
 		}
 	})
 }
+
+func TestMultipleHelloVerifyRequest(t *testing.T) {
+	// Limit runtime in case of deadlocks
+	lim := test.TimeOut(time.Second * 20)
+	defer lim.Stop()
+
+	// Check for leaking routines
+	report := test.CheckRoutines(t)
+	defer report()
+
+	cookies := make([][]byte, 1)
```

Comment:
Better to add a comment that `cookies[0]` is `nil` for the flight1 ClientHello.

Reply:
updated :+1: